editor/editor_properties.cpp

```diff
@@ -690,6 +716,7 @@ class EditorPropertyLayersGrid : public Control {
 
 	EditorPropertyLayersGrid() {
 		value = 0;
+		hovered_index = -1; // Nothing is hovered.
```

Comment:
Since the value is an `uint32_t`, this causes underflow (but the Godot builds fine). Won't this throw a warning in cppcheck and other linters though? If this is the case, I can set it to the highest possible value instead.

cc @qarmin

Reply:
Why not make `hovered_index` and `int`?

Reply:
Bump.

Reply:
Thanks, it seems to work fine when making `hovered_index` an `int`.